schemas/ticket.py

```diff
@@ -0,0 +1,33 @@
+from ma import ma
+from models.tickets import TicketModel
+from models.tenant import TenantModel
+from models.user import UserModel
+from utils.time import time_format
+from marshmallow import fields, validates, ValidationError
+
+
+class TicketSchema(ma.SQLAlchemyAutoSchema):
+    class Meta:
+        model = TicketModel
+        include_fk = True
+
+    tenant = fields.Nested("TenantSchema")
```

Comment:
It looks like the TicketModel field names should all be updated. And I don't think the Tenant has its relationship set accordingly to need this line right now, but it is OK to leave it in. I believe this is why you were seeing errors. `tenant` was already defined as a field column to hold an integer, however now it is telling Marshmallow it is a Nested field. So it was contradictory.

Reply:
Doh! Yep, yep - I see in the ticket model that we established the relationship for notes `notes = db.relationship(...)` - but not for tenant~